src/components/reusable/daterangepicker/daterangepicker.ts

```diff
@@ -300,6 +299,13 @@ export class DateRangePicker extends LitElement {
   }
 
   private validateStartEndDate(): void {
+    // Save combine values to form data
+    const formData = new FormData();
+    const dateName = this.name ?? 'date-range';
+    const combineVal = `${this.startDate} - ${this.endDate}`;
```

Comment:
Let's use something not `-` to separate the two values, since they both contain `-` already.

Reply:
As per our discussion I'd kept :